flake.nix

```diff
@@ -24,16 +22,25 @@
       # provide a dev shell for all systems that the holonix flake supports
       systems = builtins.attrNames holonix.devShells;
 
-      perSystem = { config, system, pkgs, lib, ... }:
+      perSystem = { inputs', config, system, pkgs, lib, ... }:
         {
           formatter = pkgs.nixpkgs-fmt;
 
           devShells.default = pkgs.mkShell {
-            inputsFrom = [ holonix.devShells.${system}.holochainBinaries ];
-            packages = with pkgs; [
+            inputsFrom = [ inputs'.holonix.devShells ];
+            packages = (with inputs'.holonix.packages; [
+              # add packages from Holonix
+              holochain
+              lair-keystore
+              rust
+            ]) ++ (lib.optionals pkgs.stdenv.isDarwin
+              (with pkgs.darwin.apple_sdk.frameworks; [
+                CoreFoundation
+                Security
+              ])) ++ (with pkgs; [
```

Comment:
What are the apple frameworks components needed for in the Tryorama dev shell?

Reply:
TryCP server links to it. I'd had to provide this for the build derivation previously, but I assume via the `inputsFrom` with the previous Holonix, we were importing these libraries. I got a build failure saying that `cargo build` needed to link to them -> https://github.com/holochain/tryorama/actions/runs/9911964722/job/27385772100

Reply:
Oh, I see. Cool, then it's explicit now.